src/org/zaproxy/zap/extension/bruteforce/BruteForceParam.java

```diff
@@ -37,19 +43,27 @@
 	private static final String RECURSIVE = "bruteforce.recursive";
 	private static final String BROWSE_FILES = "bruteforce.browsefiles";
 	private static final String FILE_EXTENSIONS = "bruteforce.fileextensions";
+	private static final String EXTENSIONS_TO_MISS = "bruteforce.extensionsToMiss";
+	private static final String FAIL_CASE_STRING = "bruteforce.failCaseString";
 	
 	public static final int DEFAULT_THREAD_PER_SCAN = 10;
 	public static final int MAXIMUM_THREADS_PER_SCAN = 200;
 	public static final boolean DEFAULT_RECURSIVE = true;
 	public static final boolean DEFAULT_BROWSE_FILES = false;
 	public static final String EMPTY_STRING = "";
+	public static final String DEFAULT_EXTENSIONS_TO_MISS = "jpg, gif, jpeg, ico, tiff, png, bmp";
+	public static final String DEFAULT_FAIL_CASE_STRING = Config.failCaseString;
+	public static final String WHITESPACE = "\\s";

```

Comment:
This should be private.

Reply:
@thc202 `DEFAULT_EXTENSIONS_TO_MISS`, `DEFAULT_FAIL_CASE_STRING` are also used in `OptionsBruteForcePanel.java`. Should I provide getter method or leave it public?

Reply:
Those are fine, I just mean `WHITESPACE` and `COMMA` (though it applies to `EMPTY_STRING` as well), we don't really want other classes to depend on this class just because of these generic constants.

Reply:
Sorry for the misunderstanding.
As of [this](https://github.com/zaproxy/zap-extensions/pull/1534#discussion_r180711118) comment and [this](https://softwareengineering.stackexchange.com/questions/145738/should-a-string-constant-be-defined-if-its-only-going-to-be-used-once) thread, I removed `WHITESPACE` and `COMMA` constants.

Reply:
No problem.
Sounds good.